### PR TITLE
VP-6678: Add account statuses

### PR DESCRIPTION
**Pull request by Vectorfield4**
### Problem:
No status field for Account.

### Solution:
Add a status field for Account.

### Jira:
https://virtocommerce.atlassian.net/browse/VP-6678

### CODE REVIEW

File: src/VirtoCommerce.Platform.Security/Migrations/20210120144559_Initial.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VirtoCommerce.Platform.Security.Migrations
+{
+    public partial class Initial : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
```
Comment (OlegoO) [inline src/VirtoCommerce.Platform.Security/Migrations/20210120144559_Initial.cs:9]: Why did you add this column?

Comment (Vectorfield4) [reply src/VirtoCommerce.Platform.Security/Migrations/20210120144559_Initial.cs:9]: This migration is autogenerated from `SecurityDbContextModelSnapshot`

Comment (Vectorfield4) [reply src/VirtoCommerce.Platform.Security/Migrations/20210120144559_Initial.cs:9]: @tatarincev approve or decline changes, please. 🔥

**Review by tatarincev — COMMENTED** [review]
Need to change migration according to our conversation

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-detail.js [hunk 2]
```diff
@@ -37,11 +44,11 @@ angular.module('platformWebApp').controller('platformWebApp.accountDetailControl
             return !angular.equals(blade.currentEntity, blade.origEntity) && blade.hasUpdatePermission();
         }
 
-        blade.openAccountTypeSettingManagement = function () {
+        blade.openSettingDictionaryController = function (id, currentEntityId) {
             var newBlade = {
-                id: 'accountTypesDictionary',
+                id: id,
```
[the remark this answers is not in the document]
Comment (Vectorfield4) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-detail.js:49]: I use `accountTypesDictionary` and `statusDictionary` for different blades.

[the remark this answers is not in the document]
Comment (eMazeika) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-detail.js:49]: I can see that, but id parameter is "overkill". Set is as
id: 'accountDictionarySetting',
or
id: currentEntityId,

**Comment by Vectorfield4** [comment]
### Settings
VP-6678-settings
### Creating
VP-6678-creating

**Review by akak1977 — APPROVED** [review]

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-detail.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vectorfield4) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-detail.js]: Check this out.

**Review by eMazeika — APPROVED** [review]

**Review by tatarincev — APPROVED** [review]
